Cargo.toml

```diff
@@ -63,6 +63,7 @@ smallvec = { version = "1.10", default-features = false }
 task-killswitch = { version = "0.1.0", path = "./task-killswitch" }
 thiserror = { version = "1" }
 tokio = { version = "1.44", default-features = false }
+tokio-quiche = { version = "0.5.0", path = "./tokio-quiche" }
```

Comment:
Does this need to be a workspace dep?

Reply:
Adding it here makes it consistent with other crates in the repo, so this seems fine AFAICT